tajo-common/src/main/proto/errors.proto

```diff
@@ -0,0 +1,327 @@
+  // 55 - Object Not in Prerequisite State
+  // 56 - Miscellaneous SQL or Product Error
+  // 57 - Resource Not Available or Operator Intervention
+
+  // 58 - System Error
+  IO_ERROR                      = 58030;
+
+  // 5U - Utilities Table
+
+
+  // underlying system errors based on errno.h.
+  EPERM                         = 10001;  // Operation not permitted
+  ENOENT                        = 10002;  // No such file or directory
+  ESRCH                         = 10003;  // No such process
+  EINTR                         = 10004;  // Interrupted system call
+  EIO                           = 10005;  // I/O error
+  ENXIO                         = 10006;  // No such device or address
+  E2BIG                         = 10007;  // Argument list too long
+  ENOEXEC                       = 10008;  // Exec format error
+  EBADF                         = 10009;  // Bad file number
+  ECHILD                        = 10010;  // No child processes
+  EAGAIN                        = 10011;  // Try again
+  ENOMEM                        = 10012;  // Out of memory
+  EACCES                        = 10013;  // Permission denied
+  EFAULT                        = 10014;  // Bad address
+  ENOTBLK                       = 10015;  // Block device required
+  EBUSY                         = 10016;  // Device or resource busy
+  EEXIST                        = 10017;  // File exists
+  EXDEV                         = 10018;  // Cross-device link
+  ENODEV                        = 10019;  // No such device
+  ENOTDIR                       = 10020;  // Not a directory
+  EISDIR                        = 10021;  // Is a directory
+  EINVAL                        = 10022;  // Invalid argument
+  ENFILE                        = 10023;  // File table overflow
+  EMFILE                        = 10024;  // Too many open files
+  ENOTTY                        = 10025;  // Not a typewriter
+  ETXTBSY                       = 10026;  // Text file busy
+  EFBIG                         = 10027;  // File too large
+  ENOSPC                        = 10028;  // No space left on device
+  ESPIPE                        = 10029;  // Illegal seek
+  EROFS                         = 10030;  // Read-only file system
+  EMLINK                        = 10031;  // Too many links
+  EPIPE                         = 10032;  // Broken pipe
+  EDOM                          = 10033;  // Math argument out of domain of func
+  ERANGE                        = 10034;  // Math result not representable
+;
+  EDEADLK                       = 10035;  // Resource deadlock would occur
+  ENAMETOOLONG                  = 10036;  // File name too long
+  ENOLCK                        = 10037;  // No record locks available
+  ENOSYS                        = 10038;  // Function not implemented
+  ENOTEMPTY                     = 10039;  // Directory not empty
+  ELOOP                         = 10040;  // Too many symbolic links encountered
+  // EWOULDBLOCK                = EAGAIN  // Operation would block
```

Comment:
EWOULDBLOCK and EDEADLOCK are commented out.
What is the meaning of these commented out codes? If they are TODOs, please add some comments.